src/rules/require_await.rs

```diff
@@ -163,8 +163,6 @@ fn is_body_empty(maybe_body: Option<&BlockStmt>) -> bool {
 }
 
 impl<'c, 'view> Visit for RequireAwaitVisitor<'c, 'view> {
-  noop_visit_type!();
```

Comment:
I would prefer swc be fixed to visit TS nodes in this case rather than us removing this.

Reply:
I opened https://github.com/swc-project/swc/pull/3192 but it's unclear whether it should be merged there. Perhaps we can create our own `noop_visit_type` macro that skips over types.

Reply:
@dsherret actually this rule is still using "old" visitor API instead of `deno_ast::view`. I think this problem could be fixed by rewriting the rule.